### PR TITLE
Hugged changes

### PR DESCRIPTION
**Pull request by zzzmike**
# About the pull request

Adjusts larva growth in 3 ways: eliminates growth bonus from xeno nesting, lessens protection slightly from stasis bag, stops pain and damage from larva growth while nested.

Numbers based on a config burst timer of 12 minutes:
Hugged in nest immediately -> bursts in 12 minutes, up from 8.
Hugged in stasis bag immediately -> bursts in 29.45 minutes, down from 34.54 minutes.

# Explain why it's good for the game

This is a direct result of a lot of discussion regarding "caps are goners I wish we had more time to save them" on the forums and discord. This will provide more time to get people out of nests before they burst, while not being a straight up marine buff due to stasis bags not being quite as robust as before. Being a balance change I know it will be controversial, so no hard feelings if this is rejected. I figured I might as well throw it out there and see what happens, similar to what I did with my old hugged PR https://github.com/cmss13-devs/cmss13/pull/3843 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
balance: larva growth bonus from nests removed
balance: damage/pain from larva growth now only occurs when not nested
balance: stasis bags protect you from bursting slightly less than before
/:cl:

### CODE REVIEW

**Comment by Steelpoint** [comment]
The changelog would imply stasis bags slightly speed up larva growth times.

**Comment by zzzmike** [comment]
> The changelog would imply stasis bags slightly speed up larva growth times.

Thanks, fixed

**Comment by GrrrKitten** [comment]
I feel like this is a notable buff and will just result in me fighting while hugged for a few minutes then getting evacced. I think this could be paired well with hugged marines being easier to cap (easier tackles during early stages) or something

**Comment by cuberound** [comment]
> I feel like this is a notable buff and will just result in me fighting while hugged for a few minutes then getting evacced. I think this could be paired well with hugged marines being easier to cap (easier tackles during early stages) or something

this is already in place in few forms

**Comment by zzzmike** [comment]
> I feel like this is a notable buff and will just result in me fighting while hugged for a few minutes then getting evacced.

I'm not sure what you mean. This doesn't let you fight longer when hugged, in fact it lets you fight less with bagged protection going down slightly.

**Comment by Drulikar** [comment]
Stale exempt until reviewed

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm
[the remark this answers is not in the document]
Comment (zzzmike) [reply code/modules/mob/living/carbon/xenomorph/Embryo.dm:1]: should be done

**Comment by Diegoflores31** [comment]
this is an insane xeno nerf . they loose 4 minutes on burst timer and marines pretty much nothing because they have 30 MINUTES to do surgery just for bagging

**Comment by zzzmike** [comment]
> this is an insane xeno nerf . they loose 4 minutes on burst timer and marines pretty much nothing because they still have 30 MINUTES to do surgery just for bagging

isn't explicitly 4 minutes since config can change silently at will - and I'm open to nerfing bag protection more, I dunno, what do you think @Drulikar ?

**Comment by Drulikar** [comment]
Comment was made prior to any live tm. It's in testing. So test it and get a feel for it.

**Comment by Kyogon** [comment]
This is an awful PR. It is explicitly a xeno nerf. Nerfing cap growth rate by 50% and ALSO making it so they get 50% less larva from those same caps is abysmal. The new meta will simply be to have warriors decapitate marines instead of capping since that is the only way to make sure they actually do stay dead. Doubly so when the larva from caps is halved. 

12 minutes for one larva instead of 8 minutes for about 2 is such a heavy handed nerf for xenos.

**Comment by zzzmike** [comment]
> This is an awful PR. It is explicitly a xeno nerf. Nerfing cap growth rate by 50% and ALSO making it so they get 50% less larva from those same caps

Not explicitly a xeno nerf as it nerfs stasis bag protection as well (and I might tune the numbers to nerf bags slightly more if Drathek is onboard tbh)

> ALSO making it so they get 50% less larva from those same caps

This PR does not touch larva per cap, where are you getting this from?

**Comment by Drulikar** [comment]
I'm not going to be available much this month to observe this in game, feel free to adjust the values as you think better suits what you are trying to do and or work with another maintainer to fine tune it like Zone. But also obviously the entire burst time is a config too; so ultimately what needs to be established here is the ratio between being in a cryo bag or not. Note that any changes made to a PR though a maintainer has to explicitly update the TM to reflect those changes in game.

**Comment by zzzmike** [comment]
Sounds good, thanks Drulikar.

Anyways, so far from what I've seen, xenos don't seem to struggling. Have observed quick xeno wins (wiping most marines out sub-1 hour) and also xeno victories in the long haul like in 23345, a 3-hour round (which would be affected more than short rounds by the longer nested time). That being said, I don't have log access so I don't have the best picture when it comes to seeing how this PR is affecting things.

**Comment by Git-Nivrak** [comment]
If this gets merged you probably want to nerf anti-parasitic as it was buffed due to burst time being reduced to 7.5 minutes previously

**Comment by zzzmike** [comment]
> If this gets merged you probably want to nerf anti-parasitic as it was buffed due to burst time being reduced to 7.5 minutes previously

I'm not familiar with the situation, but, I would hope that someone makes it so that it just affects each larva growth stage by a percentage, that way it'll scale with any PR which affects burst time.

**Comment by HeresKozmos** [comment]
This PR is straight up a xeno nerf.

**Comment by Bidux27** [comment]
If non-mantainers are not allowed to comment here flag this as off-topic.

The lower larva timer was set up not long ago because of the change giving only one larva per cap and also so that you would feel the need of getting surgery when hugged instead of shrugging it off as a mild inconvenience. This nerf not only undoes some of that balance but also makes it so that if a hugged marine dies while nested it is basically certain that they will not burst, instead of previously when you still had a grace period. 
Survs and monkeys are not even bursted when marines land and if the hive is in a corner with marines at its doors, the act of capping almost does nothing as they will never burst in time to make a difference, which is one of the most fun situations of the game in my opinion.

My voice does not count at all here, because it shouldn't, but I see this change as creating way more collateral damages than the problem it tries to solve.

**Comment by Drulikar** [comment]
> If non-mantainers are not allowed to comment here flag this as off-topic.
Comments on PRs on github should be pertaining to the code. General feedback goes here: https://forum.cm-ss13.com/w/pr-feedback/steps/step_1

**Comment by zzzmike** [comment]
> This nerf not only undoes some of that balance

This change does not undo the urgency to get surgery at all. The only time your burst is slower is if you're nested, and if you're nested you can't seek surgery anyways. With the stasis bag protection going down slightly, the need is actually more pressing.

>  also makes it so that if a hugged marine dies while nested it is basically certain that they will not burst

This is why pain and damage from larva while nested was eliminated in this PR. And it does seem to make a difference for people surviving until burst from what I've seen.

**Comment by Nanu308** [comment]
In it's current state personally not a fan of it wit how heavily it impacts xeno and larva timer, which was reduced from how lengthy it was in the past with good reason.

**Comment by harryob** [comment]
pr is stale for a couple of weeks with no movement on unblocking it. let us know if you want this reopened if you work it out with nanu

**Pull request closed without merging**